core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -6726,6 +6756,87 @@ components:
           $ref: '#/components/schemas/ProjectRole'
           description: the role of the user in the project
       description: an element of a list of project users and their role
+    BillingProjectAdminResponse:
+      required:
+        - billingProject
+        - workspaces
+      type: object
+      properties:
+        billingProject:
+          $ref: '#/components/schemas/RawlsBillingProject'
+        workspaces:
+          type: object
+          additionalProperties:
+            type: string
+            format: uuid
+          example:
+            "workspace1": "00000000-0000-0000-0000-000000000000"
+            "workspace2": "11111111-1111-1111-1111-111111111111"
+          description: "workspaces in the billing project with their IDs"
+    RawlsBillingProject:
+      required:
+        - projectName
+        - status
+        - invalidBillingAccount
+      type: object
+      description: internal representation of a billing project, returned by admin API only
+      properties:
+        projectName:
+          type: string
+          description: the name of the project
+        billingAccount:
+          type: string
+          description: the billing account to use in google projects
+        invalidBillingAccount:
+          type: boolean
+          description: whether or not the billing account is usable by Terra
+        status:
+          type: string
+          enum:
+            - Creating
+            - Ready
+            - Error
+            - Deleting
+            - DeletionFailed
+            - AddingToPerimeter
+            - CreatingLandingZone
+          description: the status of allocating the billing project's resources.
+        message:
+          type: string
+          description: informational message about the project
+        azureManagedAppCoordinates:
+          $ref: '#/components/schemas/AzureManagedAppCoordinates'
+        cloudPlatform:
+          type: string
+          enum:
+            - GCP
+            - AZURE
+            - UNKNOWN
+        landingZoneId:
+          type: string
+          format: uuid
+          description: the UUID of the landing zone associated with the project (cloudPlatform AZURE only)
+        billingProfileId:
+          type: string
+          description: the billing profile ID associated with the project
+        cromwellBackend:
+          type: string
+          description: the cromwell backend to use for this billing project
```

Comment:
Can there be a more useful description? I didn't know we had more than one Cromwell backend.

Reply:
I gave this vague description because I don't have a good understanding of what it's for. It's not set by default. I found a comment that says that possible values are "JES" and "PAPIv2", but I'm not sure if that's still accurate.

Reply:
Ah, it's very old then!

Why can't you use the existing `RawlsBillingProjectResponse`? Are they too different?

Reply:
Yeah, they're just a bit different so I thought it was better to document what can actually get returned